content/docs/2.2/scalers/openstack-metric.md

```diff
@@ -0,0 +1,174 @@
+---
+title: "Openstack Metric"
+date: 2021-03-07T17:14:29-03:00
+draft: true
+---
++++
+title = "Openstack Metric"
+layout = "scaler"
+availability = "v2.2+"
+maintainer = "Community"
+description = "Scale applications based on the metrics collected from an specific resource."
+go_file = "openstack_metric_scaler"
++++
+
+### Trigger Specification
+This specification describes the `openstack-metric` trigger for OpenStack metrics based on gnocchi API. It scales based on an spcific measeure from a given resource metric.
+
+```yaml
+triggers:
+- type: openstack-metric
+  metadata:
+    metricsURL: localhost:8041/v1/metric #required
+    metricID: 003bb589-166d-439d-8c31-cbf098d863de #required
+    aggregationMethod: "mean" #required
+    granularity: 300 #required (seconds)
+    threshold: 1250 #required
+    timeout: 30 #optional
+```
+
+> Please, always provide the protocol (http or https) when specifying URLs. This is needed due to Go URL parsing issues :sweat_smile:
+
+**Parameter list:**
+- `metricsURL` - The URL to query the metrics gnocchi API. It must contain the hostname, the gnocchi port, the API version, and the resource ID. The pattern is: `http://<host>:<swift_port>/<swift_version>/<resource_id>` 
```

Comment:
Is `gnocchi` required here?

Reply:
Hi, I updated the docs about gnocchi. Actually gnocch is not required, its an openstack embedded time series database aimed to store and make it ease to request metrics which provides an API with an standard, the version here is linked with gnocchi api version, but it depends on openstack configurations and what api version it is providing.